### PR TITLE
Improving the appearance of the BitDataGrid component example in the BlazouUI demo project (#2718)

### LINKED ISSUE #2718: Inappropriate demo page of the `BitDataGrid` component
The samples in the demo page of the `BitDataGrid` component needs to be modified just like the [Radzen](https://blazor.radzen.com/datagrid) samples. Also, It's better to use the `BitSpinner` component for data loading in the samples.

### PR DESCRIPTION
**Pull request by MeysamBahadori**
This closes #2718

### CODE REVIEW

**Review by ysmoradi — CHANGES_REQUESTED** [review]

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor [hunk 1]
```diff
@@ -5,25 +5,28 @@
...
                             <BitSearchBox @bind-Value="typicalSampleNameFilter"
                                           Placeholder="Search on Name"
                                           InputHtmlAttributes="@(new Dictionary<string, object> {{"autofocus", true}})" />
                         </ColumnOptions>
                     </BitDataGridPropertyColumn>
-                    <BitDataGridPropertyColumn Property="@(c => c.Medals.Gold)" Sortable="true" Align="BitDataGridAlign.Right" />
-                    <BitDataGridPropertyColumn Property="@(c => c.Medals.Silver)" Sortable="true" Align="BitDataGridAlign.Right" />
-                    <BitDataGridPropertyColumn Property="@(c => c.Medals.Bronze)" Sortable="true" Align="BitDataGridAlign.Right" />
-                    <BitDataGridPropertyColumn Property="@(c => c.Medals.Total)" Sortable="true" Align="BitDataGridAlign.Right" />
+                    <BitDataGridTemplateColumn Title="Flag" Align="BitDataGridAlign.Center">
```
[the remark this answers is not in the document]
Comment (MeysamBahadori) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor:23]: All these fields are sortable

**Review by msynk — APPROVED** [review]

**Comment by msynk** [comment]
@maryamhdr 
plz review

**Review by ysmoradi — DISMISSED** [review]

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor.cs [hunk 2]
```diff
@@ -287,50 +285,146 @@ public partial class BitDataGridDemo
...
 <style scoped>
 .grid {
-    display: inline-flex;
-    flex-direction: column;
-    width: 100%;
+  display: inline-flex;
+  flex-direction: column;
+  width: 100%;
+  border: 1px solid #e9eaed;
+}
```
Comment (maryamhdr) [inline src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor.cs:292]: plz add a blank line after declaring each CSS class block.

**Review by maryamhdr — CHANGES_REQUESTED** [review]

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor.cs (hunk 2 above)
Comment (MeysamBahadori) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor.cs:293]: ok

**Review by maryamhdr — APPROVED** [review]

**Pull request merged**